compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -351,9 +365,14 @@ impl Inliner<'tcx> {
         if callee_body.basic_blocks().len() <= 3 {
             threshold += threshold / 4;
         }
+
+        // Give a huge bonus to private functions called only once in current module.
+        if self.tcx.mir_inliner_callers_count_map(()).get(&callsite.callee.def_id()) == Some(&1) {
```

Comment:
Likewise, we should use some kind of  `mir_inliner_callers_count(callside.callee.def_id())` here.